packit_service/worker/whitelist.py

```diff
@@ -191,95 +301,45 @@ def check_and_report(
         :param job_configs: iterable of jobconfigs - so we know how to update status of the PR
         :return:
         """
+        # Administrators
+        user_login = getattr(event, "user_login", None)
+        if user_login and user_login in service_config.admins:
+            logger.info(f"{user_login} is admin, you shall pass.")
+            return True
 
-        # whitelist checks dont apply to CentOS (Pagure, Gitlab)
-        if isinstance(
-            event,
+        for callback, related_events in (
```

Comment:
What about some mapping, this is becoming too magical to me.

Reply:
it is a mapping, without map :D i'll add it

Reply:
My point was more about readability. Just that it's really clear what is going on here.